Modules/Core/Common/include/itkCovariantVector.hxx

```diff
@@ -113,7 +113,7 @@ template <typename T, unsigned int VVectorDimension>
 typename CovariantVector<T, VVectorDimension>::ValueType
 CovariantVector<T, VVectorDimension>::operator*(const Vector<T, VVectorDimension> & other) const
 {
-  typename NumericTraits<T>::AccumulateType value = T{};
+  auto value = T{};
```

Comment:
You're changing the type of `value`, from `AccumulateType` to `T`. Is that intended? Otherwise, you might do:

    typename NumericTraits<T>::AccumulateType value{};